installer.sh

```diff
@@ -67,19 +67,22 @@ if [[ -n "$(command -v dnf)" ]]; then
     PYTHON_DEPS="python3-pip gcc gcc-c++ openssl-devel swig python3-pyyaml python3-m2crypto python3-tornado python3-simplejson python3-requests yaml-cpp-devel procps-ng"
     PYTHON_PIPS="pycryptodomex tornado pyzmq"
     BUILD_TOOLS="openssl-devel libtool make automake pkg-config m4 libgcrypt-devel autoconf autoconf-archive libcurl-devel libstdc++-devel uriparser-devel dbus-devel gnulib-devel doxygen"
+    TPM2_SOFTWARE="tpm2-tss tpm2-tools tpm2-abrmd"
 elif [[ -n "$(command -v yum)" ]]; then
     PACKAGE_MGR=$(command -v yum)
     $PACKAGE_MGR -y install epel-release
     PYTHON_PREIN="python36 python36-devel python36-setuptools python36-pip git wget patch openssl"
     PYTHON_DEPS="gcc gcc-c++ openssl-devel swig python36-PyYAML python36-tornado python36-simplejson python36-requests yaml-cpp-devel"
     PYTHON_PIPS="pycryptodomex tornado pyzmq m2crypto"
     BUILD_TOOLS="openssl-devel libtool make automake m4 libgcrypt-devel autoconf autoconf-archive libcurl-devel libstdc++-devel uriparser-devel dbus-devel gnulib-devel doxygen"
+    TPM2_SOFTWARE="tpm2-tss tpm2-tools tpm2-abrmd"
 elif [[ -n "$(command -v apt-get)" ]]; then
     PACKAGE_MGR=$(command -v apt-get)
     PYTHON_PREIN="git patch"
     PYTHON_DEPS="python3 python3-pip python3-dev python3-setuptools python3-zmq python3-tornado python3-simplejson python3-requests gcc g++ libssl-dev swig python3-yaml wget"
     PYTHON_PIPS="pycryptodomex m2crypto"
     BUILD_TOOLS="build-essential libtool automake pkg-config m4 libgcrypt20-dev uthash-dev autoconf autoconf-archive libcurl4-gnutls-dev gnulib doxygen libdbus-1-dev"
+    TPM2_SOFTWARE="tpm2-tss tpm2-tools tpm2-abrmd"
```

Comment:
I don't think these versions support the features we need (our Keylime-specific modifications). 

For tpm2-tools, our changes were released in [3.2](https://github.com/tpm2-software/tpm2-tools/releases/tag/3.2.0) and are also in the latest/master/4.X, while even the latest available in Ubuntu is 3.1.x. 

This will break Ubuntu below, since we're removing the possibility to build from source.

Reply:
Hm, good point. I am just really not into compiling code in an installer and who knows how long it will take for Debian to package the right version.

Have you checked against disco / buster?

Reply:
https://packages.debian.org/buster/tpm2-tools

Reply:
Can we just build from source if they're using apt, otherwise use the package manager?